internal/fxlog/fxlog_test.go

```diff
@@ -123,6 +123,15 @@ func TestPrint(t *testing.T) {
 		assert.Equal(t, "", sink.String())
 	})
 
+	t.Run("printStripsVendorPath", func(t *testing.T) {
+		sink.Reset()
+		// assert is vendored within fx and is a good test case
+		logger.PrintProvide(assert.New)
+		assert.Contains(
+			t, sink.String(),
+			"*assert.Assertions <= vendor/github.com/stretchr/testify/assert.New()")
```

Comment:
For example, the `dig` package can be present in the vendor folder, when it is installed by e.g. glide or it can be taken from `%GOPATH%/src/go.uber.org/dig` folder if it is fetched by `go get` before. In both cases project will be compiled successfully.